test/units/module_utils/facts/test_date_time.py

```diff
@@ -7,29 +7,45 @@
 
 import pytest
 import datetime
+import time
 
-from ansible.module_utils.facts.system.date_time import DateTimeFactCollector
+from ansible.module_utils.facts.system import date_time
 
-TIMESTAMP = datetime.datetime(2020, 7, 11, 12, 34, 56, 124356)
+EPOCH_TS = 1594449296.123456
```

Comment:
```suggestion
EPOCH_TS = 1594449296.124356
```

Reply:
I'm not sure I understand why this change is necessary. Could you enlighten me? 😄

Reply:
Looks like values below in `DT` and `DT_UTC` are 124356, instead of 123456.

Reply:
Originally, there was `124356` microseconds in unit tests, instead of `123456`. At least two tests are rely on this figure.